### ISSUE TITLE
enable setting config from file and add compile instructions

### ISSUE THREAD

**Issue by egonw** [bridgedb/Wikidata2Bridgedb#4]
And it's only from within Eclipse right now, right?

**Comment by mkutmon** [comment]
yes currently only in eclipse - need to change to provide parameters for output dir, old db, new db name - will work on that over the weekend.

**Comment by egonw** [comment]
I added some pointers to the readme.
